core/design/src/main/kotlin/edu/stanford/spezi/core/design/theme/Borders.kt

```diff
@@ -0,0 +1,8 @@
+package edu.stanford.spezi.core.design.theme
+
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+object Borders {
```

Comment:
that sounds good and we can do it like that. i have only included the elevations in Sizes now, because here i would assume, in contrast to possibly ratios, that this is needed again in many places